### PR TITLE
task(config): add android pass through config option ProjectPackages

### PR DESCRIPTION
**Pull request by rich-bugsnag**
## Goal

add android pass through config option ProjectPackages

## Changeset

- Added ProjectPackages to config class
- Pass the value through to android on init

## Testing

Manually tested, CI tests also added

### CODE REVIEW

**Comment by twometresteve** [comment]
Why aren't CI tests possible?  I can see that we can't check the value set in the Android layer directly, but can't we infer that it's been set correctly by having a scenario that:
- Sets a single ProjectPackage
- Causes a native error
- Verifies that a stack frame is marked as inProject and another not

If the ProjectPackage is set differently to what the default would be, that should be sufficient to infer that the config has taken effect.

**Comment by rich-bugsnag** [comment]
@twometresteve Added tests 👍

File: features/fixtures/maze_runner/Assets/Scripts/MobileScenarioRunner.cs [hunk 1]
```diff
@@ -168,6 +174,11 @@ public void DoTestAction(string scenarioName)
                 SetUser();
                 LogError();
                 break;
+            case "Java Background Crash":
+                LeaveBreadcrumbString();
+                LeaveBreadcrumbTuple();
```
Comment (lemnik) [inline features/fixtures/maze_runner/Assets/Scripts/MobileScenarioRunner.cs:179]: Given the separation of "throw Exception" and "throw Exception with breadcrumbs": should "Java Background Crash" really be leaving breadcrumbs before crashing?

Comment (rich-bugsnag) [reply features/fixtures/maze_runner/Assets/Scripts/MobileScenarioRunner.cs]: Removed the unnecessary calls to the breadcrumb methods

**Review by lemnik — APPROVED** [review]
LGTM

**Pull request merged**